thirdeye/thirdeye-frontend/app/mocks/alertConfig.js

```diff
@@ -0,0 +1,60 @@
+export default [
+  {
+    "id": 1,
```

Comment:
since this is just a Javascript object, the keys property name does not need quotation marks `""`. 
applies to whole pr